engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/types/EspTypeUtilsSpec.scala

```diff
@@ -187,7 +186,8 @@ class EspTypeUtilsSpec extends FunSuite with Matchers {
     final val quxDocs = "This is sample documentation for qux method"
   }
 
-  test("should extract description and params from method") {
+  //FIXME: scala 2.11, 2.12 have different behaviour...
```

Comment:
How we will live with it? Is scala 2.12 build is still usable without this fix?

Reply:
I added more accurate description, and ignore only two problematic cases, we can try to fix it in next PR